addons/mail/static/tests/mail_test_helpers_contains.js

```diff
@@ -430,6 +432,27 @@ export async function insertText(selector, content, options = {}) {
     await animationFrame(); // wait for t-model synced with new value
 }
 
+/**
+ * @param {Editor} editor
+ * @param {string} text
+ * @param {Object} [options]
+ * @param {boolean} [options.replace=false] whether to replace the current editor content with the text
+ */
+export async function htmlInsertText(editor, text, options = {}) {
+    if (options.replace) {
+        editor.editable.innerHTML = "<div class='o-paragraph'><br></div>";
+        await manuallyDispatchProgrammaticEvent.silent(editor.editable, "keydown", {
+            key: "Backspace",
+        });
+        await manuallyDispatchProgrammaticEvent.silent(editor.editable, "keyup", {
+            key: "Backspace",
+        });
+        await manuallyDispatchProgrammaticEvent.silent(editor.editable, "input");
+        setSelection({ anchorNode: editor.editable, anchorOffset: 0 });
+    }
+    await htmlInsert(editor, text);
+}
```

Comment:
`mail_test_helpers_contains` is not the right file for this method.

`mail_test_helpers_contains` is a legacy and basically deprecated file, and all methods in this file takes a selector as first param and wait for it to be present in the DOM.

This should be moved to another file, `mail_test_helpers` if anything.